src/MetricQWebView.js

```diff
@@ -218,6 +216,26 @@ class MetricQWebView {
     }, this.RELOAD_THROTTLING_DELAY)
   }
 
+  async addMetric (metricBase, description = undefined, oldMetric = undefined) {
+    try {
+      await this.store.dispatch('metrics/create', { metric: { ...oldMetric, name: metricBase, description: description, traces: [] } })
```

Comment:
Now the boolean return value of `addMetric` is used, but a bool is never returned here.